core/src/main/java/com/microsoft/applicationinsights/internal/config/PerformanceCountersXmlElement.java

```diff
@@ -32,6 +30,7 @@
 @XmlRootElement(name="PerformanceCounters")
 public class PerformanceCountersXmlElement {
     private boolean useBuiltIn = true;
+    private long cadenceInSec = 60;
```

Comment:
You already defined a constant with the default value. It's better if you can use this constant everywhere instead of writing 60 in two different places